docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: cloud
+tags: 
+ - devops
+ - role:cloud-engineer
+ - size:small
+---
+
+# Leverage automation to start/stop instances and re-create non-production environments
+
+## Description
+
+Non-production workloads take up significant computing resources
+and adds to the carbon footprint of your application.
+
+Non-productions workloads, like development and test environments, need not be running all the time (like weekends,non-business hours, and extended non-utilization periods) and can be optimized through automation for efficient utilization.
```

Comment:
This seems to be duplicating an existing pattern in the catalog: https://patterns.greensoftware.foundation/catalog/cloud/minimizing-deployed-environments

Are there specific differences you want to call out in this proposal or can we update that pattern or mark this as a duplicate?

Reply:
Great! I'll mark this PR as closed then and wait for the new one with the updates to the existing pattern. Thanks!